apps/festival/festival/src/app/dashboard/home/home.component.ts

```diff
@@ -70,6 +99,16 @@ export class HomeComponent {
     map(analytics => analytics.filter(analytic => analytic.name === 'pageView'))
   );
 
+  activeCountries$ = this.titleAnalytics$.pipe(
+    filter(analytics => analytics.length > 0),
+    map(analytics => {
+      const orgs = analytics.map((analytic) => analytic.org)
+      return { orgs, analytics };
+    }),
+    map(getTerritoryAndCount),
+    map(stats => stats.sort(({ count: countA }, { count: countB }) => countA - countB))
```

Comment:
You're making it more complicated than need be
```suggestion
    map(stats => stats.sort((a, b) => a.count - b.count))
```